source/common/http/conn_manager_config.h

```diff
@@ -435,6 +436,11 @@ class ConnectionManagerConfig {
    */
   virtual envoy::config::core::v3::HttpProtocolOptions::HeadersWithUnderscoresAction
   headersWithUnderscoresAction() const PURE;
+
+  /**
+   * @return LocalReply configuration which supplies mapping for local reply generated by Envoy.
+   */
+  virtual const LocalReply::LocalReply* localReply() const PURE;
```

Comment:
nit: `absl::optional<std::reference_wrapper<...>>` see typedef style we do elsewhere.

Reply:
Done